docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -0,0 +1,61 @@
+# ECE340
+
+## Content Covered
+
+- Band diagrams
+- Fermi-Dirac/Maxwell-Boltzmann Statistics
+- Drift and Diffusion of Carriers
+- PN Junctions
+- MOSCAPs and MOSFETs
+- BJTs
+- Optoelectronics (Photodetectors, Solar Cells, LEDs, LASERs)
```

Comment:
Preferably format like this
https://github.com/hkn-alpha/wiki/pull/6#discussion_r1251461683